source/tunneling/README.md

```diff
@@ -55,6 +56,34 @@ $ ./localproxy -r us-east-1 -s 8080 -t <source token>
 $ ssh -p 8080 <remote_user_name>@localhost
 ```
 
+### Policy Permissions
+In order to use the Secure Tunneling feature the device must first have permission to connect to IoT Core. 
+The device must also be able to subscribe to and receive messages on the Secure Tunneling notify topic in order to receive the destination token.
+The example policy below demonstrates the minimum permissions required. Simply replace the `<region>` and `<accountId`> with the correct values.
```

Comment:
Can you update this line to -> `The example policy below demonstrates the minimum permissions required to run Secure Tunneling feature.`